Data_Structures_Questions.md

```diff
@@ -3,51 +3,56 @@ Answer the following questions for each of the data structures you implemented a
 ## Queue
 
 1. What is the runtime complexity of `enqueue`?
-
+   O(1)
 2. What is the runtime complexity of `dequeue`?
-
+   O(1)
 3. What is the runtime complexity of `len`?
+   O(n)
 
 ## Binary Search Tree
 
-1. What is the runtime complexity of `insert`? 
-
+1. What is the runtime complexity of `insert`?
```

Comment:
This is the brilliance of a binary search tree; that ability to cut down on half of the data that needs to be parsed on each move down the tree.